satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -1,8 +1,18 @@
-// dbx.v1 golang satellitedb.dbx .
+// dbx.v1 golang irreparabledb.dbx .
```

Comment:
Let's separate these into sections, so it's clear which things are related. For example:
```
// datarepair.IrreparableDB

model irreparabledb (
```

Reply:
Done